src/dpq2/conv/native_tests.d

```diff
@@ -46,7 +46,17 @@ public void _integration_test( string connParam ) @system
 
                 auto answer2 = conn.execParams(params);
                 auto v2 = answer2[0][0];
-                auto textResult = v2.as!string.strip(' ');
+
+                string textResult;
+
+                {
+                    auto str = v2.as!(Nullable!string);
+                    if(str.isNull)
+                        textResult = "NULL";
+                    else
+                        textResult = str.get.strip(' ');
+                }
+
```

Comment:
What about keeping it simple:
```D
auto textResult = v2.isNull ? null : v2.as!string.strip(' ');
```

And let the test keep using `null` instead of `"NULL"`

Reply:
Because we can confuse textResult == null what means empty string with `NULL` as PG value
It is better to make such things more obvious.

Reply:
Still can be
```D
auto textResult = v2.isNull ? "NULL" : v2.as!string.strip(' ');
```

then